doc/dev/mempool_accounting.rst

```diff
@@ -0,0 +1,51 @@
+What is a mempool?
+-----------------
+A memory pool is a method for accounting the consumption of memory of
+a set of containers.
+Memory pools are statically declared (see pool_index_t) Ultimately, which serves
+the purpose of identifying memory related checks in bluestore.
+
+Some common mempools that we can track
+--------------------------------------
+
+- bloom_filter: Ceph uses bloom filters to account for tracking objects already
+  in the cache pool, in order to determine which objects in the pool are being
+  accessed.
+- bluestore_alloc: It accounts for actual allocations for bitmap allocator or
```

Comment:
In fact this applies would apply to any allocator implementation including avl and hybrid ones. But it looks like we do not use mempools there which is apparently a bug to be fixed. Hence suggest not to mention specific allocator types here